### ISSUE TITLE
🐛  Type error when dragging columns under group by state

### ISSUE THREAD

**Issue by mujuni88** [inovua/reactdatagrid#212]
**Version:**
`"@inovua/reactdatagrid-enterprise": "^4.1.24" `

**Steps**

Pre-requisite:
* Set `disableGroupByToolbar = true` on `ReactDataGrid`
* Set `groupBy=column`. Make sure you are in **group by** state

**Problem:**
When you drag columns around, you will receive the following error:
```
Uncaught TypeError: Cannot read properties of null (reading 'getCells')
 at InovuaDataGridHeaderLayout.setupDrag (index.js:940)
 at onMouseMove (index.js:746)
```

The reason it returns `null` is because on `setupDrag`, you call `this.getGroupToolbar().getCells()` which will not work since we `disableGroupByToolbar = true`. The issue goes away when you set `disableGroupByToolbar = false`

**Comment by kevin-hi** [comment]
Also seeing the same error, above essentially renders impossible to drag/reorder columns while SortBy is active.

**Comment by inovua-admin** [comment]
Fixed in `v4.19.5`.
